### ISSUE TITLE
Discuss for kl_ in loss function

### ISSUE THREAD

**Issue by quangnhien** [kumar-shridhar/PyTorch-BayesianCNN#26]
I see you compute kl_ in your code like that:
kl_ = math.log(self.q_logvar_init) - self.sigma_weight + (sig_weight**2 + self.mu_weight**2) / (2 * self.q_logvar_init ** 2) - 0.5.
I can't map the above formulate with anything what in your paper.
Can you show me why you use this formulate?

Please help me! :(.
Thanks

**Comment by kumar-shridhar** [comment]
Hi,

This is how the KL divergence is calculated: https://stats.stackexchange.com/questions/7440/kl-divergence-between-two-univariate-gaussians
Hope this helps.

**Comment by quangnhien** [comment]
Thank for your answer very much. I already understand.
This mean you chosse distribution of prior as N(0,0.05).
You have a great paper.
